lib/sidekiq/delay_extensions/generic_proxy.rb

```diff
@@ -10,7 +10,7 @@ class Proxy < BasicObject
       def initialize(performable, target, options = {})
         @performable = performable
         @target = target
-        @opts = options
+        @opts = options.transform_keys(&:to_s)
```

Comment:
this isn't a great idea since it's not part of how sidekiq used to work and in any case won't work with how the keys are sent.

Reply:
Perhaps some egg on my face. This was added in Sidekiq 6.4.2.  In any case, it's now in the last 6.x version of this gem and in the new 7.0 release